pkg/tsdb/cloudwatch/models/settings_test.go

```diff
@@ -1,16 +1,23 @@
 package models
 
 import (
+	"context"
 	"testing"
 	"time"
 
 	"github.com/grafana/grafana-aws-sdk/pkg/awsds"
 	"github.com/grafana/grafana-plugin-sdk-go/backend"
+	"github.com/grafana/grafana-plugin-sdk-go/backend/proxy"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
 func Test_Settings_LoadCloudWatchSettings(t *testing.T) {
+	settingCtx := backend.WithGrafanaConfig(context.Background(), backend.NewGrafanaCfg(map[string]string{
```

Comment:
This context just exists so that the tests not testing the context settings don't produce a bunch of warning messages